pandas/core/indexing.py

```diff
@@ -43,7 +43,37 @@ def get_indexers_list():
 
 # the public IndexSlicerMaker
 class _IndexSlice(object):
-
+    """
+    Create an object to more easily perform multi-index slicing
+
+    Examples
+    --------
+
+    >>> miindex = pd.MultiIndex.from_product([['A0','A1'],['B0','B1','B2','B3']])
```

Comment:
This line is too long. But you can rename the `miindex` to eg `midx`, then it should fit :-)